### ISSUE TITLE
gym version conflict between genrl and pettingzoo

### ISSUE THREAD

**Issue by veds12** [SforAiDl/genrl#385]
genrl 0.0.2 requires gym 0.17.1 whereas pettingzoo 1.3.3 requires gym  0.17.2

**Comment by veds12** [comment]
@Sharad24

**Comment by Sharad24** [comment]
Can you run tests with 0.17.2? I don't really see a reason to stick with an older gym version

**Comment by veds12** [comment]
Yeah. A bunch of stuff is outdated actually including pytorch and tensorboard. I'll run the the tests for newer versions of all of them

**Comment by Sharad24** [comment]
Sg!
